app/grandchallenge/evaluation/tasks.py

```diff
@@ -263,6 +267,24 @@ def create_algorithm_jobs_for_evaluation(*, evaluation_pk, first_run):
         )
 
     if first_run:
+        # Only go ahead if this is the users only active evaluation to
+        # prevent flooding of the system
+        user_has_other_active_evaluations = (
+            Evaluation.objects.filter(
+                submission__creator=evaluation.submission.creator,
+                status=Evaluation.EXECUTING_PREREQUISITES,
```

Comment:
Why only filter on that specific status here and not use `.active()`?

Reply:
Because the other steps we can do in parallel without issue.